src/article.ts

```diff
@@ -220,11 +219,14 @@ const articleFieldsWithBody = (docParser: DocParser, content: Content): ArticleF
         body: parseElements(docParser)(content.blocks.body[0].elements),
     });
 
+const containsOpinionTags = (tags: Tag[]): boolean =>
+    tags.some(tag => tag.id === 'tone/comment' || tag.id === 'tone/letters')
+
 const fromCapi = (docParser: DocParser) => (content: Content): Article => {
-    switch (content.type) {
+    const { type, tags, pillarId, fields, blocks } = content;
+    switch (type) {
```

Comment:
Slightly cautious about this just because `type` is a reserved word in TypeScript.